From 34ac8a4f31f490db5cec7b6291dc98479c45c78f Mon Sep 17 00:00:00 2001
From: Christian Hesse <mail@eworm.de>
Date: Fri, 1 Mar 2019 10:00:27 +0100
Subject: [PATCH 1/1] systemd service: add instance name in description

The unit files made systemd print:

systemd[1]: Started MariaDB 10.3.13 database server (multi-instance).

Let's add the instance name, so starting mariadb@foo.service
makes it print:

systemd[1]: Started MariaDB 10.3.13 database server (multi-instance foo).

Signed-off-by: Christian Hesse <mail@eworm.de>
---
 support-files/mariadb@.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/support-files/mariadb@.service.in b/support-files/mariadb@.service.in
index 727c1b85301..fd997b2ce3d 100644
--- a/support-files/mariadb@.service.in
+++ b/support-files/mariadb@.service.in
@@ -131,7 +131,7 @@
 # systemd.service(5)
 
 [Unit]
-Description=MariaDB @VERSION@ database server (multi-instance)
+Description=MariaDB @VERSION@ database server (multi-instance %I)
 Documentation=man:mysqld(8)
 Documentation=https://mariadb.com/kb/en/library/systemd/
 After=network.target
